### PR TITLE
[Readme] Reformatting, add "Contributing" and "Help" sections

### PR DESCRIPTION
**Pull request by johnsaigle**
### Brief summary of changes

Made a bunch of changes to the presentation of the Readme. This is obviously very subjective so let me know what you think.

I mainly tried to reduce duplication, change the format to highlight the most important areas, and encourage readers to open issues and reach out to us generally.

Also removed the build status chart and replaced it with the `master` build status which was placed beside the title.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -89,6 +86,21 @@ LORIS requires Apache's mod_rewrite module to rewrite its URLs. Enable this modu
...
-# Community
-Please feel free to subscribe to the [LORIS Developers mailing list](http://www.bic.mni.mcgill.ca/mailman/listinfo/loris-dev) to ask any LORIS-related questions.
+## Help
+
+### Create an Issue
+Please don't hesitate to create an issue if you're stuck with something. Please
+include details such as the version of LORIS you're using as well as information
+such as the OS you're using, your PHP and Apache versions, etc.
+
+### Community
```
Comment (driusan) [inline README.md]: Community isn't a subset of help. Help is a subset of community.

File: README.md [hunk 2]
```diff
@@ -1,43 +1,40 @@
...
+#### System Requirements
 
- * LINUX (supported on Ubuntu 14+ and [CentOS 6.5](https://github.com/aces/Loris/blob/master/README.CentOS6.md))
+ * Linux (supported on Ubuntu 14+ and [CentOS 6.5](https://github.com/aces/Loris/blob/master/README.CentOS6.md))
  * Apache2
  * MySQL 5.7 
  * PHP <b>7.2</b>  
- * Package manager (for LINUX distributions)
- * Composer : should be run with --no-dev option
+ * A package manager, e.g. `apt`
```
Comment (PapillonMcGill) [inline README.md]: Apache 2.4
MySQL 5.7+ or MariaDB 10.3+
PHP 7.2 or higher

We had a discussion on that in https://github.com/aces/Loris/pull/3976

File: README.md [hunk 3]
```diff
@@ -1,43 +1,40 @@
...
-([CentOS Readme also available](https://github.com/aces/Loris/blob/master/README.CentOS6.md)).
 
-Please consult the [LORIS Wiki Setup Guide](https://github.com/aces/Loris/wiki/Setup) notes on this [Install process](https://github.com/aces/Loris/wiki/Installing-Loris) for more information not included in this Readme. The [LORIS Developers mailing list](http://www.bic.mni.mcgill.ca/mailman/listinfo/loris-dev) may also provide installation guidance not covered in the Wiki. 
+## Installation
 
-# Prerequisites for Installation
+#### System Requirements
 
- * LINUX (supported on Ubuntu 14+ and [CentOS 6.5](https://github.com/aces/Loris/blob/master/README.CentOS6.md))
+ * Linux (supported on Ubuntu 14+ and [CentOS 6.5](https://github.com/aces/Loris/blob/master/README.CentOS6.md))
```
Comment (PapillonMcGill) [inline README.md]:
```suggestion
 * Linux (supported on Ubuntu 14+ and [CentOS 6.5+](https://github.com/aces/Loris/blob/master/README.CentOS6.md))
```

File: README.md [hunk 4]
```diff
@@ -1,40 +1,40 @@
...
 ([CentOS Readme also available](https://github.com/aces/Loris/blob/master/README.CentOS6.md)).
 
-Please consult the [LORIS Wiki Setup Guide](https://github.com/aces/Loris/wiki/Setup) notes on this [Install process](https://github.com/aces/Loris/wiki/Installing-Loris) for more information not included in this Readme. The [LORIS Developers mailing list](http://www.bic.mni.mcgill.ca/mailman/listinfo/loris-dev) may also provide installation guidance not covered in the Wiki. 
+Please consult the [LORIS Wiki Setup Guide](https://github.com/aces/Loris/wiki/Setup) notes on this [Install process](https://github.com/aces/Loris/wiki/Installing-Loris) for more information.
+
+#### Heroku
+
+You can try LORIS on Heroku before installing it on your system. The project management and clinical data management functions of LORIS are available for experimenting with. Imaging functionality is not yet available.
+
+Deploy and log in with username <i>admin</i> and the password that's set up during deployment via ClearDB.
```
Comment (driusan) [inline README.md]:
```suggestion
Deploy and log in with username *admin* and the password that's set up during deployment via ClearDB.
```

Change from HTML to markdown.

File: README.md [hunk 5]
```diff
@@ -86,6 +86,21 @@ LORIS requires Apache's mod_rewrite module to rewrite its URLs. Enable this modu
...
+Please feel free to subscribe to the [LORIS Developers mailing list](http://www.bic.mni.mcgill.ca/mailman/listinfo/loris-dev) to ask any LORIS-related questions. We may also be able to provide you with installation guidance not covered in the Wiki.
+
+## Contributing
+
+### Contributing Code
+If you would like to contribue to LORIS development, please consult our
+[Contributing Guide](./CONTRIBUTING.md).
 
+### Contributing Documentation
+We love suggestions for our documentation! If you notice any inaccuracies, typos, etc. please open a GitHub Issue and we'll address it.
```
Comment (driusan) [inline README.md]:
```suggestion
```

This is out of place, it should be in the `CONTRIBUTING.md` which is referenced directly above it.

File: README.md [hunk 6]
```diff
@@ -86,6 +86,20 @@ LORIS requires Apache's mod_rewrite module to rewrite its URLs. Enable this modu
...
+### Get in touch
+Please feel free to subscribe to the [LORIS Developers mailing list](http://www.bic.mni.mcgill.ca/mailman/listinfo/loris-dev) to ask any LORIS-related questions. We may also be able to provide you with installation guidance not covered in the Wiki.
+
+## Contributing
+
+### Contributing Code
+If you would like to contribue to LORIS development, please consult our
+[Contributing Guide](./CONTRIBUTING.md).
 
+### Contributing Documentation
```
Comment (driusan) [inline README.md]:
```suggestion
```

This heading doesn't have a body (the previous suggestion could only suggest one line..)

Comment (johnsaigle) [reply README.md]: Yeah I really hope they implement multi-line suggestions soon...

**Comment by PapillonMcGill** [comment]
would it be possible in step 4 of install,
to replace this line:
`sudo service apache2 reload`
with:
`sudo systemctl reload apache2`
to take advantage of SystemD tools?

Or would it be for another PR?

**Comment by johnsaigle** [comment]
I don't really know much about the difference between the two. From [this post](https://askubuntu.com/a/903405) it seems like `service` might be more helpful because it does some extra checks for us.

What's is your preference for using the lower-level command over this one?

**Comment by PapillonMcGill** [comment]
service is the old form. With systemD, systemctl is the new way to do it. If you call service with SystemD, a wrapper will pass the call to systemctl for backward compatibility (not exhaustive).

Lower level command is systemctl as no wrapper is involved

**Comment by johnsaigle** [comment]
Apart from one being old and another new I'm still not sure why we would need to change the command here. `service` seems to be a wrapper for different init systems and is smart enough to redirect to `systemctl` when appropriate + does error-handling. I'm not sure why we'd change to something more specific and therefore potentially compatible with fewer environments.

Anyway I think this is probably a topic for a different PR since this is mostly just a formatting change.

File: README.md [hunk 7]
```diff
@@ -1,40 +1,40 @@
-# LORIS Neuroimaging Platform
+# [![Build Status](https://travis-ci.org/aces/Loris.svg?branch=master)](https://travis-ci.org/aces/Loris) LORIS Neuroimaging Platform 
 
-LORIS (Longitudinal Online Research and Imaging System) is a web-based data and project management software for neuroimaging research. LORIS makes it easy to manage large datasets including behavioural, clinical, neuroimaging and genetic data acquired over time or at different sites.
+LORIS (Longitudinal Online Research and Imaging System) is a self-hosted web application that provides data- and project-management for neuroimaging research. LORIS makes it easy to manage large datasets including behavioural, clinical, neuroimaging and genetic data acquired over time or at different sites.
```
Comment (christinerogers) [inline README.md]: This paragraph is extremely bolded by this PR.

**Pull request merged**